src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -13,13 +13,19 @@ def main():
     endf_path = 'tape20'
     pendf_path = 'tape21'
 
-    material_id, MTs = tp.extract_endf_specs(endf_path)
+    material_id, MTs, endftk_file_obj = tp.extract_endf_specs(endf_path)
```

Comment:
could you find the intersection of `MTs` and `mt_dict.keys()` here to make your life simpler?

Reply:
I suppose I could, although I'm not sure that it offers much benefit as the intersection is found in line 94 of `groupr_tools.py`, with the line `for MT in MTs & mt_dict.keys():`, which will only pass the acceptable MT values for the NJOY run, so only MT values in that intersection will appear in the processed GENDF file.

Reply:
The benefit is that you restrict the set of MT numbers as soon as possible.  This can prevent confusion about which MT numbers are included in the list at which points in the code.  If you had a reason to keep the full list of MT numbers for some other operation, that would support making this down-selection as late as possible.